css/component_styles/image.css

```diff
@@ -0,0 +1,18 @@
+/* Responsive Image CSS */
+.rsp-img{
+    width: 100%;
+    height: auto;
+    object-fit: cover;
+}
+.image-frame{
+    height: 27rem;
+}
+
+/* Round Image */
+.round-img{
+    height: 150px;
+    width: 150px;
```

Comment:
if possible use relative units

Reply:
Done,
Change it to rem relative unit.
thanks for the suggestion👍🏻